### ISSUE TITLE
[BUG] Alpha Chanel in PNGs for iOS

### ISSUE THREAD

**Issue by elzahaby** [fluttercommunity/flutter_launcher_icons#375]
### :information_source: Info

Version: 0.9.3

### :speech_balloon: Description

The app icons created for iOS seem to always have the alpha Chanel enabled when using PNGs, which is not allowed for App Store upload. The Icon must have no transparency.

### :scroll: Pubspec.yaml

```yaml
name: achievy
description: Achievy helps you beating procrastination.

publish_to: 'none' # Remove this line if you wish to publish to pub.dev
version: 1.0.2+6

environment:
  sdk: ">=2.15.1 <3.0.0"

dependencies:
  flutter:
    sdk: flutter

  cupertino_icons: ^1.0.2

dev_dependencies:
  flutter_test:
    sdk: flutter

  flutter_lints: ^1.0.0
  flutter_launcher_icons: "^0.9.2"


flutter_icons:
  ios: true
  image_path: "assets/icon/icon.png"

flutter:

  uses-material-design: true
```

**Comment by RatakondalaArun** [comment]
Hi @elzahaby You can disable the alpha channel with `remove_alpha_ios`

```yaml
flutter_icons:
  ios: true
  image_path: "assets/icon/icon.png"
  remove_alpha_ios: true # disables alpha channel
```

we need to update docs

**Comment by RatakondalaArun** [comment]
This issue is resolved in `v0.9.0`
closing this as duplicate of #253

**Comment by solomonium** [comment]
I added that flag to disable the alpha but still got same error.. how else can I go about it please
